database/schema.sql

```diff
@@ -0,0 +1,23 @@
+CREATE SCHEMA IF NOT EXISTS homes2;
+
+DROP TABLE IF EXISTS homes2.homes;
+
+CREATE TABLE homes2.homes (
+  id SERIAL PRIMARY KEY, 
+  zestimationPrice TEXT,
+  thirtyDayPriceChange TEXT,
+  oneYearForcast TEXT,
+  comparableHomePrice TEXT,
+  marketAppreciationPrice TEXT,
+  url TEXT,
+  sellDate TEXT,
+  sellPrice TEXT,
+  beds INT,
+  baths INT,
+  streetAddress TEXT,
+  priceSqft TEXT,
+  saleToList TEXT
+);
+
+COPY homes2.homes(id,zestimationPrice,thirtyDayPriceChange,oneYearForcast,comparableHomePrice,marketAppreciationPrice,url,sellDate,sellPrice,beds,baths,streetAddress,priceSqft,saleToList) 
+FROM '/Users/pony/SEC/home-value/sdcDataGeneration/primaryentries19.csv' DELIMITER ';' CSV;
```

Comment:
so I was wondering how to do this to be able to allow another person to simply run this file without them having to make any changes, wouldn't this only work for you since path is specific to your computer setup?

Reply:
I'm actually not certain. This would only work on my machine with my current file path...